main.py

```diff
@@ -1,7 +1,63 @@
 import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
```

Comment:
Consider refactoring the creation of `Genre` and `Actor` instances using a list of tuples to avoid repetitive code.